style.css

```diff
@@ -515,3 +515,423 @@ footer {
+    margin-bottom: 26px;
+  }
+
+  .headline {
+    width: 100%;
+    height: 798px;
+    background-image: url(images/desk-img1.png);
+    background-repeat: no-repeat;
+    background-position: right;
+    background-size: contain;
+  }
+
+  .headline-bigger {
+    width: 566px;
+    margin-left: 143px;
+    margin-top: 60px;
+    order: 1;
+  }
+
+  .headline-text {
+    width: 566px;
+    font-weight: 500;
+    font-size: 32px;
+    margin-left: 143px;
+    margin-top: 24px;
+    order: 3;
+  }
+
+  .headline-small-text {
+    width: 566px;
+    height: 123px;
+    font-weight: 500;
+    font-size: 20px;
+    margin-left: 143px;
+    margin-top: 24px;
+    order: 4;
+  }
+
+  .social-icons {
+    width: 20px;
+    height: 20px;
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    left: 0.5%;
+    right: 96.11%;
+    top: 25.37%;
+    bottom: 89.19%;
+    order: 2;
+    margin-top: 140px;
+    margin-left: 36px;
+    margin-right: 90px;
+  }
+
+  .background-img {
+    display: none;
+  }
+
+  .recent {
+    display: flex;
+  }
+
+  .recent-work {
+    width: 315px;
+    font-family: 400;
+    font-size: 40px;
+    margin-right: 24px;
+  }
+
+  .line {
+    width: 794px;
+    height: 0;
+    border: 1px solid #212121;
+    margin-top: 130px;
+  }
+
+  .under-page {
+    width: 1156px;
+    height: 446px;
+    display: flex;
+  }
+
+  .first-img {
+    display: none;
+  }
+
+  .under-img {
+    display: block;
+  }
+
+  .under-text {
+    margin-left: 24px;
+  }
+
+  .pages-container {
+    display: grid;
+    grid-template-columns: repeat(3, 369px);
+    grid-template-rows: repeat(2, 435px);
+    justify-items: center;
+    width: 100%;
+    column-gap: 24px;
+    row-gap: 37px;
+    padding-left: 120px;
+    margin-left: 50px;
+  }
+
+  .page {
+    display: none;
+  }
+
+  .page-header {
+    display: none;
+  }
+
+  .page-paragraph {
+    display: none;
+  }
+
+  .page-language {
+    display: none;
+  }
+
+  .page-see-project {
+    display: none;
+  }
+
+  .page1 {
+    width: 369px;
+    height: 435px;
+    display: block;
+    background-image: url(images/desk-pages1.png);
+  }
+
+  .page1-button {
+    width: 369px;
+    height: 58px;
+    display: inline-block;
+    margin-top: 377px;
+    text-align: center;
+  }
+
+  .page2 {
+    width: 369px;
+    height: 435px;
+    display: block;
+    background-image: url(images/desk-pages2.png);
+  }
+
+  .page2-header {
+    display: flex;
+    width: 337;
+    height: 81px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+    padding-left: 13px;
+    margin-top: 178px;
+  }
+
+  .page2-paragraph {
+    display: flex;
+  }
+
+  .page2-lang {
+    display: flex;
+  }
+
+  .page3 {
+    width: 369px;
+    height: 435px;
+    display: flex;
+    background-image: url(images/desk-pages3.png);
+  }
+
+  .page3-header {
+    width: 337;
+    height: 38px;
```

Comment:
- [ ] Kindly add the unit to your width.

Reply:
- [ ] Make sure to add the required unit to your width as suggested by the previous reviewer. The width property should always be accompanied by a unit such as `px`, `rem`, `% `